README.md

```diff
@@ -1,23 +1,52 @@
+### Policies
+The MacAdmins Slack Team is dedicated to providing a harassment-free experience for everyone, regardless of (but not limited to): gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem. 
 
-If you have concerns regarding this Slack instance please feel free to contact the administrative team and we will work to understand and address them respectfully and with an appropriate level of privacy to the given situation.
+We do not tolerate harassment of Slack participants in any form. Sexual language and imagery is not appropriate for any venue. Slack participants violating these rules may be sanctioned or expelled from the MacAdmins Slack at the discretion of the Administrative Team. 
 
-We hope that you find the community to be enriching to your personal and professional lives, and above all else a place of camaraderie where our collective knowledge and differences are celebrated in a positive, welcoming environment.
+Harassment includes, but is not limited to:
 
-Thank you for following this code of conduct. We reserve the right to amend or change the code of conduct at any time and encourage you to periodically review these guidelines to ensure a safe environment for all.
+* Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Hate speech
+* Sea-lioning ([definition](http://simplikation.com/why-sealioning-is-bad/))
+* Gas-lighting ([definition](https://en.wikipedia.org/wiki/Gaslighting))
+* Spamming of GIFs
+* Advocating for, or encouraging, any of the above behaviour	
 
--MacAdmins Slack Administrative Team 
+
+### Enforcement
+Participants asked to stop any harassing behavior are expected to comply immediately
+
+If a participant engages in harassing behaviour, the Administrative Team retain the right to take any actions to keep the event a welcoming environment for all participants. This includes warning the offender or expulsion from the MacAdmins Slack.
+
+The Administrative Team may take action to redress anything designed to, or with the clear impact of, disrupting the event or making the environment hostile for any participants.
+
+We expect participants to follow these rules in all public and private channels (including direct messages).
+
+### Reporting
+Reporting violations of these rules should be done directly to a member of the Administrative Team.
```

Comment:
Reporting violations of these rules should be done directly to a member of the Administrative Team. Either by contacting them in #ask-the-slack-admins channel, via direct message, or via email listed on their Slack profile which can be found on the [Current List of Administrators](https://github.com/macadminsdotorg/slack-assets/blob/master/Admins)